### PR TITLE
TASK: Make post articleBody less wide for better readability

### PR DESCRIPTION
**Pull request by danielkestler**

### CODE REVIEW

**Comment by Sebobo** [comment]
Hi @danielkestler,

thanks for improving this. Just a nitpick after testing, can you fix the indentation in the scss file?
Then I can merge it.

Next time please do the PR from a separate branch and allow maintainers to edit it, then we can quickly fix that before merging.

File: Packages/Sites/Neos.NeosIo/Resources/Private/Scss/Organisms/_Post.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.postBody {
+    max-width: $layout-small-width;
```
Comment (Sebobo) [inline Packages/Sites/Neos.NeosIo/Resources/Private/Scss/Organisms/_Post.scss]: Indentation doesn't match here.

**Comment by danielkestler** [comment]
Sorry, now indentation should match.

**Comment by kdambekalns** [comment]
@danielkestler could you please resolve the merge conflict!? Thanks!

**Comment by Sebobo** [comment]
Merged this in #200

**Pull request closed without merging**